debian/rules

```diff
@@ -4,11 +4,18 @@ export DEB_BUILD_OPTIONS=nocheck
 #export DH_VERBOSE = 1
 
 %:
-ifneq ($(shell dh -l | grep -xF translations),)
-	dh $@ --with python3,translations --buildsystem=pybuild
-else
 	dh $@ --with python3 --buildsystem=pybuild
-endif
+
+override_dh_auto_build:
+	python3 -m pip wheel --no-build-isolation --no-deps .
```

Comment:
When I tested replacing `python setup.py build` with this on my Arch system it gets significantly slower: 
<img width="2276" height="1116" alt="image" src="https://github.com/user-attachments/assets/5f42d7d6-7315-47c4-8247-8bb940a1838b" />